client/reader/following-manage/index.jsx

```diff
@@ -137,6 +137,24 @@ class FollowingManage extends Component {
 		);
 	};
 
+	reportFollowByUrlRender = siteUrl => {
+		if ( siteUrl ) {
+			recordTrack( 'calypso_reader_following_manage_follow_by_url_render', {
+				url: siteUrl,
+			} );
+		}
+	};
+
+	shouldShowFollowByUrl = () => resemblesUrl( this.props.sitesQuery );
+	componentDidUpdate() {
+		if (
+			this.shouldShowFollowByUrl
```

Comment:
also need to verify that the url changed from the last update and only report when it changes. and add reporting to componentDidMount, as componentDidUpdate doesn't fire during the initial mount.